README.md

```diff
@@ -1,10 +1,27 @@
 OpenReview Python library
 =========================
 
-
 [![CircleCI](https://circleci.com/gh/iesl/openreview-py.svg?style=svg)](https://circleci.com/gh/iesl/openreview-py)
 [![Documentation Status](https://readthedocs.org/projects/openreview-py/badge/?version=latest)](https://openreview-py.readthedocs.io/en/latest/?badge=latest)
 
+Prerequisites
```

Comment:
if you want to merge the README file in another PR then you need to remove the changes of this file in this PR.